src/form/internal.tsx

```diff
@@ -19,34 +20,53 @@ export default function InternalForm({
   errorIconAriaLabel,
   actions,
   secondaryActions,
+  variant = 'full-page',
```

Comment:
If the `full-page` is the "default", should we put this at the `index.jsx` level to reflect it in the API documentation?

Reply:
Good catch! Didn't realize it needed to be separate for API docs to recognize it. Updated!